scripts/add-task.js

```diff
@@ -14,18 +14,17 @@ function addTask(
     if (form.className == HIDE_CLASS) {
       form.classList.remove(HIDE_CLASS);
     }
-    inputValue.focus();
+    inputElementValue.focus();
   });
 
   cancelButton.addEventListener(CLICK_EVENT, () => {
     form.classList.add(HIDE_CLASS);
-    inputValue.value = null;
+    inputElementValue.value = null;
```

Comment:
accessing the `value` property of a variable named `inputElementValue`, does it look logical?

```suggestion
    inputElement.value = null;
```